receiver/receivercreator/README.md

```diff
@@ -67,18 +69,25 @@ extensions:
   k8s_observer:
 
 receivers:
+  prometheus_simple:
+    # Configure prometheus scraping if standard prometheus annotations are set on the pod.
+    rule: type.pod && annotations["prometheus.io/scrape"] == "true"
+    config:
+      metrics_path: '`"prometheus.io/path" in annotations ? annotations["prometheus.io/path"] : "/metrics"`'
+      endpoint: '`endpoint`:`"prometheus.io/port" in annotations ? annotations["prometheus.io/port"] : 9090`'
```

Comment:
Looks like it's put under a wrong `receivers` section. should be under `receiver_creator/1` -> `receivers`. right?

Reply:
Yep, thanks